Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1346,6 +1346,10 @@
 		F9E197D32283D05C0092B3E1 /* CrashLogging.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = CrashLogging.swift; sourceTree = "<group>"; };
 /* End PBXFileReference section */
 
+/* Begin PBXFileSystemSynchronizedRootGroup section */
+		3FA8C2862DFBAF7900E1F4A3 /* WordPressRatings */ = {isa = PBXFileSystemSynchronizedRootGroup; explicitFileTypes = {}; explicitFolders = (); path = WordPressRatings; sourceTree = "<group>"; };
+/* End PBXFileSystemSynchronizedRootGroup section */
+
```

Comment:
Notice I imported the files via a synchronized folder. Everything new that we bring into our Xcode projects should be in synchronized folders IMHO. A more light weight way to track files in the project, with far less changes of merge conflicts when new files are added/renamed/removed.